manifests/init.pp

```diff
@@ -7,10 +7,13 @@
 # $port::             port of foreman proxy that is used as $server
 #
 # $ca_file::          path to file of certification authority that issued client's certificate
+#                     May be overriden if $::rh_certificate_repo_ca_file (from Facter) is found
```

Comment:
why not provide the default in the class signature vs introduce another parameter name?
i think the following should work:

```puppet
class foreman_scap_client($ca_file =  $::rh_certificate_repo_ca_file,...
```

however, when i think about it a bit more, not sure if we should simply present a params class since you want to fall back to puppet certs?

any other ideas? @mmoll @ekohl @domcleal?